package.json

```
@@ -2,19 +2,19 @@
   "name": "@side/npm-dependency-stats-action",
   "version": "0.0.0-development",
   "description": "Github action for getting stats about npm dependencies",
+  "author": "Side Inc.",
+  "license": "MIT",
   "main": "dist/index.js",
   "scripts": {
     "format": "prettier --write **/*.ts",
     "format-check": "prettier --check **/*.ts",
     "lint": "eslint src/**/*.ts",
-    "lint:fix": "npm run lint -- --fix",
+    "lint:fix": "yarn lint -- --fix",
```

```suggestion
    "lint:fix": "yarn lint --fix",
```

However, is this one really needed to begin with?  `yarn lint:fix` vs `yarn lint --fix`

---

I find it nice to have personally, but I could see the argument for removing